config/version-compatibility-matrix.yaml

```diff
@@ -26,82 +26,58 @@
       kialiMinimumVersion: "1.18.1"
       kialiMaximumVersion: "1.21"
     - meshVersion: "1.5"
-      kialiMinimumVersion: "1.17"
-      kialiMaximumVersion: "1.17"
+      kialiFixedVersion: "1.17"
     - meshVersion: "1.4"
-      kialiMinimumVersion: "1.17"
-      kialiMaximumVersion: "1.17"
+      kialiFixedVersion: "1.17"
     - meshVersion: "1.3"
-      kialiMinimumVersion: "1.17"
-      kialiMaximumVersion: "1.17"
+      kialiFixedVersion: "1.17"
     - meshVersion: "1.2"
-      kialiMinimumVersion: "1.17"
-      kialiMaximumVersion: "1.17"
+      kialiFixedVersion: "1.17"
     - meshVersion: "1.1"
-      kialiMinimumVersion: "1.17"
-      kialiMaximumVersion: "1.17"
+      kialiFixedVersion: "1.17"
     - meshVersion: "1.0"
-      kialiMinimumVersion: "1.17"
-      kialiMaximumVersion: "1.17"
+      kialiFixedVersion: "1.17"
 
 # The Compatible Version Matrix between Maistra and Kiali (there is a 1-to-1 relationship between Kiali and Maistra versions)
 - meshName: "Maistra"
   versionRange: 
     - meshVersion: "2.2"
-      kialiMinimumVersion: "1.48"
-      kialiMaximumVersion: ""
+      kialiFixedVersion: "1.48"
     - meshVersion: "2.2"
-      kialiMinimumVersion: "1.36"
-      kialiMaximumVersion: ""
+      kialiFixedVersion: "1.36"
     - meshVersion: "2.2"
-      kialiMinimumVersion: "1.24"
-      kialiMaximumVersion: ""
+      kialiFixedVersion: "1.24"
```

Comment:
This is better but do we really need the multiple entries for the, in this case, the 2.2 meshVersion?  I mean, if you're running 2.2 you need 1.48, not 1.36 or 1.24.  Or, is this somehow because the 1.48 operator installs 2.2, 2.1 and 2.0?

Reply:
> Or, is this somehow because the 1.48 operator installs 2.2, 2.1 and 2.0?

That how I see it. If you are OSSM 2.2, you could be running Kiali Server 1.24. (OSSM 2.2 operator can install Kiali 1.24).

Reply:
I confirmed jay is right - we only need one entry. I included that change in my PR to this PR.